bridgetown-core/lib/bridgetown-core.rb

```diff
@@ -250,6 +250,12 @@ def sanitized_path(base_directory, questionable_path)
       end
     end
 
+    def configure
+      s = Struct.new(:foo).new
+      yield s
+      puts s
+    end
```

Comment:
```ruby
Bridgetown.configure do |config|
  config.foo = "Bar"
end
#<struct foo="Bar">
```
Now I can receive args in my `Bridgetown.configure` I'm wondering how can I receive `n` args without naming it.